src/vs/base/common/product.ts

```diff
@@ -91,6 +91,8 @@ export interface IProductConfiguration {
 		readonly productName: string;
 	};
 
```

Comment:
separate cleanup task, can we put all telemetry related stuff next to each other (the tas stuff above is related to both experiments and telemetry so it can be moved after experiments url). Then we can move all the other telemetry stuff next to and after that